README.md

```diff
@@ -9,36 +9,86 @@ To use the generic wallet refer to `signTransaction.py`, `getPublicKey.py` or Le
 # How to Install developer version
 ## Configuring Ledger Environment
 
-* Install Vagrant and Virtualbox on your machine
-* Run the following
+* Install Docker on your machines
+* Checkout the app builder repository
 
+Note the `./full` path on `docker build` differentiate from the previous version now under `legacy`
 ```
-git clone https://github.com/fix/ledger-vagrant
-cd ledger-vagrant
-vagrant up
+git clone https://github.com/LedgerHQ/ledger-app-builder.git
+cd ledger-app-builder
+sudo docker build -t ledger-app-builder:latest ./full
 ```
 
 This will take a few minutes to install
 
+## Prepare to Connect Device
+Set `udev` rules to enable devices to connect with docker container. *Note:* Instructions are for linux and tested on Ubuntu 22.
+```
+wget -q -O - https://raw.githubusercontent.com/LedgerHQ/udev-rules/master/add_udev_rules.sh | sudo bash
+```
+
+Download your app
+```
+git clone https://github.com/eosnetworkfoundation/ledger-app.git
+```
+
 ## Compile your ledger app
 
-* install your app under apps/ for instance:
+* go to your instance directory:
+* copy over the files and enter the docker container
+```
+cd ledger-app
+sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
```

Comment:
I would have:
- add the --user option
- use the image from ghcr.io/ledgerhq/
- use the `lite` image `ledger-app-builder-lite`

Reply:
quite the same on all your docker commands below.